Source/cryptography/implementation/OpenSSL/Vault.cpp

```diff
@@ -420,4 +423,24 @@ uint32_t netflix_security_wrapping_key(void)
     return (Implementation::Vault::NetflixInstance().Size(Implementation::Netflix::KPW_ID) != 0 ? Implementation::Netflix::KPW_ID : 0);
 }
 
+uint32_t persistence_key_exists( struct VaultImplementation* vault ,const string& locator,bool& result)
```

Comment:
Can we have this coherent? We have IPersistent, persistent_implementation.h, so perhaps the namespace prefix could also be persistent_...?